js/components/stories/Breakpoints.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+const React = require("react");
+const { DOM: dom, createElement } = React;
+const { storiesOf } = require("@kadira/storybook");
+const configureStore = require("../../create-store");
+const { combineReducers } = require("redux");
+const reducers = require("../../reducers");
+const { Provider } = require("react-redux");
+const { fromJS } = require("immutable");
+
+const Breakpoints = React.createFactory(require("../Breakpoints"));
+
+const state = {
+  sources: {
+    "fooSourceActor": {
+      actor: "fooSourceActor",
+      url: "http://example.com/foo.js"
+    },
+    "barSourceActor": {
+      actor: "barSourceActor",
+      url: "http://example.com/bar.js"
+    }
+  },
+  breakpoints: {
+    "fooBreakpointActor": {
+      actor: "fooBreakpointActor",
+      location: {
+        actor: "fooSourceActor",
+        line: 16
+      },
+    },
+    "barBreakpointActor": {
+      actor: "barBreakpointActor",
+      location: {
+        actor: "barSourceActor",
+        line: 18
+      },
+    }
+  }
+};
+
+storiesOf("Breakpoints", module)
+  .add("No Breakpoints", () => {
+    const store = configureStore({})(combineReducers(reducers), {
```

Comment:
I like this as documentation. It would be nice to make this as clear as possible though if it's going to be used as component docs. What if you made `renderBreakpoints` take the state instead, so none of these stories would to do the `configureStore` stuff? All it would do is pass the state.